16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316cd50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b02d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab51d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab5250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab6510>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac3350>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac5ad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a97410>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316ff50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0e310>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2c850>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a7ac10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2fe90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0de10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1079e4cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2d0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6e510>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0f250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b950>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b38b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b39c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b48690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b50bd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b3b250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1d3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ce50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0cc50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b3a750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b510>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac5250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a78b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac6910>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316cd90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1fe50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2d910>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac7990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a94ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2d750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6e0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac6890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b627d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107adf6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a790d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a97390>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107add050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6f850>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b62650>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a94b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107870610>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316f350>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac24d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abc290>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316ef50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1e210>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b60bd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1e390>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107884cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b534d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0db10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b49050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac5a90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2d890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a79c50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b38d50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abcad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4a250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abf490>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abd190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b48c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b39c90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2fc90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1d750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac6210>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b50b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2dc90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac0210>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac7b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0c250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2fd50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1078878d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a96d10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b150>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0e190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107885950>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e790>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b61590>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac56d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b78650>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107adc610>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac3a50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0e050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac0510>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107885c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0f590>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b52b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b0690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b04dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b53890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1c210>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b0310>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316f410>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a94310>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1fdd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a78910>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a96dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1e690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a7ac10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316fd90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac2890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2cbd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b39010>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abe5d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107869990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1ebd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b3a090>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1c0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1df90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1079d2ad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1fed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316e8d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b3a050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2f850>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abcad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2dd50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316df90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abc6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abead0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ea10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316fad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b78390>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107871f90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ed50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2fc50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abcd90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abcb90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2cfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ea10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abf410>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab5e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6d350>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107886650>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b0450>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b0310>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107add490>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6dd50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b0310>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7a3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab54d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b52890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e210>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107adf810>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1fbd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab5290>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316fed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab5ad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e7d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1df90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abd710>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab53d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ec90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b50790>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0d590>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0c2d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b04a50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10556c690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1079d2b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05010>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b38b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0efd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1f750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b04490>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2dfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1ef10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b609d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a79250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07e90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b61f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05f90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ed90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ea10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b79b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abe3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b60410>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b51fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06d50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac6250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a96e90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b61dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab7190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e110>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab75d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b602d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b616d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2c190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab6fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a7b810>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a95250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a6cf10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab6890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1f9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab4cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316c090>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac2710>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7a810>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b1a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac4190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac1050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac6e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b49490>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316e710>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac1990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b78f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0c590>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac1990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b50b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac1050>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b059d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7b8d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1f110>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316e690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b789d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7bad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1c850>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b067d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7b150>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b63d10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1d090>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1031b12d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2d650>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab6b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10316ea90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab4d50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac2710>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b290>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab6ad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4b990>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b60610>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a974d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107885a50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab7c90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2df10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b61c50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e4d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2e790>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b48610>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1ec50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b3a3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2c8d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07490>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b524d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107885010>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1de50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac1a90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1030a4b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b7bb90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab4b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac7ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a7af90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac0a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b0e4d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06d50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b39d90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ac3f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2ef50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b05f90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2c550>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b07bd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b62010>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b383d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b48290>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b04710>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b616d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b4a890>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107abc690>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06750>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a945d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b63a50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b06fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107a97250>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b9bf90>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab4090>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b1fe10>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:02 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b853d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:03 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105487190>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:03 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ab5c50>: Failed to establish a new connection: [Errno 61] Connection refused'))
16 11:17:03 - An error occurred: HTTPConnectionPool(host='localhost', port=55566): Max retries exceeded with url: /session/eebc80ec2bd65bbe568ccd7b575592c0/window/handles (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107b2c050>: Failed to establish a new connection: [Errno 61] Connection refused'))
